cicecore/drivers/unittest/sumchk/sumchk.F90

```diff
@@ -5,11 +5,12 @@ program sumchk
       use CICE_FinalMod
       use ice_kinds_mod, only: int_kind, dbl_kind, real_kind
       use ice_communicate, only: my_task, master_task, get_num_procs
+      use ice_domain_size, only: nx_global, ny_global
       use ice_domain_size, only: block_size_x, block_size_y, max_blocks
       use ice_domain, only: distrb_info
       use ice_blocks, only: block, get_block, nx_block, ny_block, nblocks_tot
       use ice_distribution, only: ice_distributionGetBlockID, ice_distributionGet
-      use ice_constants, only: field_loc_center
+      use ice_constants, only: field_loc_Nface
```

Comment:
I'm not sure I understand why this is changed here...

Reply:
sumchk is testing the ice_global_reduction infrastructure.  If you look at that infrastructure, it relies heavily on the information contained in the grid and decomposition including the location and size of the halo, whether the grid is a tripole grid, and so forth.  For non-tripole grids, the global reductions operate on the grids (sans halo areas) and the field_loc value makes no difference.  However, for the tripole grids, there is special logic for field_loc_Nface and field_loc_NEface in order to remove redundant points on the tripole zipper.  

I have added a test of sumchk with the tx1 grid and in order to trigger the tripole logic in the global reduction, I have to set the field location as Nface or NEface.  I picked Nface arbitrarily.  This has no impact on non-tripole grids.  If I stick with "center" for tripole grids, I don't trigger the tripole logic.  One could argue I should test center and Nface for tripole and non-tripole grids, but I decided to just do Nface for both.  Maybe I'll revise that.